.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Template for requesting new features
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+## Context
+
+What is the problem that you are trying to solve?
+What did you try, and why it didn't work?
+Why is this problem relevant?
+
+## Solution
+
+(This section and the ones below are optional.)
+What are the possible solutions?
+If there are multiple alternatives, what are the benefits and drawbacks of each one?
+
+## Subtasks
+
+- [ ] If there is a solution, what are the subtasks for completing this issue?
+
+## Definition of Done
```

Comment:
I think this "Definition of done" section could be removed from feature-request template.

Reply:
i think it makes sense to keep it  
features may be defined in different ways and DoD may facilitate the understanding of the particular feature request

Reply:
This is also marked as optional.